name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages_de.properties

```diff
@@ -2201,15 +2201,15 @@ PresetsPrefPageTime = Uhrzeit
 
 PresetsPrefPageTitle = Voreinstellungen
 
-RebalanceAmbiguousTooltip = Dieses Rebalancing Ergebnis ist mehrdeutig.\n\nDurch Verfeinern der Taxonomie oder durch Ausschlie\u00DFen von Wertpapieren vom Rebalancing kann die Mehrdeutigkeit behoben werden.
+RebalanceAmbiguousTooltip = Dieses Rebalancing-Ergebnis ist mehrdeutig.\n\nDurch Verfeinern der Klassifizierung oder durch Ausschlie\u00DFen von Wertpapieren vom Rebalancing kann die Mehrdeutigkeit behoben werden.
 
-RebalanceInexactTooltip = Dieses Rebalancing Ergebnis ist nicht exakt, weil es keine exakte L\u00F6sung gibt.\n\nDiese L\u00F6sung minimiert den mittleren quadratischen Fehler unter der Nebenbedingung, dass die Rebalancing Werte sich zu 0 addieren m\u00FCssen.\n\nDurch vergr\u00F6bern der Taxonomie oder durch Einschlie\u00DFen von weiteren Wertpapieren beim Rebalancing kann eine genaue L\u00F6sung erm\u00F6glicht werden.
+RebalanceInexactTooltip = Dieses Rebalancing-Ergebnis ist nicht exakt, weil es keine exakte L\u00F6sung gibt.\n\nDiese L\u00F6sung minimiert den mittleren quadratischen Fehler unter der Nebenbedingung, dass die Rebalancingwerte sich zu 0 addieren m\u00FCssen.\n\nDurch Vergr\u00F6bern der Klassifizierung oder durch Einschlie\u00DFen von weiteren Wertpapieren beim Rebalancing kann eine genaue L\u00F6sung erm\u00F6glicht werden.
 
-SecuritiesChart_NoDataMessage_NoHoldings = Keine Best\u00E4nde vorhanden
+SecuritiesChart_NoDataMessage_NoHoldings = Das Wertpapier wurde nie gehalten
```

Comment:
The point isn‘t that there are presently “keine Bestände”, but that there never has been a period where the security was held.